README.md

```diff
@@ -1 +1,8 @@
-# myre2
\ No newline at end of file
+# myre2
+
+
+def myfunc(lis):
+    lis.sort()
```

Comment:
Do not use built-in functions. Idea is you understand the how to loop through list to find a maximum value. It covers for loop and if  statement to check.